Controller/Main.as

```diff
@@ -209,6 +214,17 @@ void RefreshLeaderboard(){
                     alreadyHandled = medalFound[medalIndex];
                 }
             }
+
+            // Update positions for club members
+            for (uint j = 0; j < clubEntries.Length; j++) {
+                if (clubEntries[j].time == resp.positions[i].time) {
+                    resp.positions[i].positionData = clubEntries[j].positionData;
```

Comment:
were you able to test the edge case where two club members had the same time? I think you might get the same position data in that case.
If I'm right (it's been a long weekend for me x)), you might need something like an array of bool (the length of clubentries) then say that the J entry has been found when you do find it, then skip the entry for future loops